<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameStoreGUIClass</class>
 <widget class="QMainWindow" name="GameStoreGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GameStoreGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_Login">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>330</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Войти</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_UserName">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>210</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Логин</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Password">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>280</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Пароль</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_UserName">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>21</y>
      <width>88</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Balance">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>21</y>
      <width>88</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lst_Games">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>256</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Registration">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>380</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Регистрация</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_AddToFavorites">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить в избранное</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Buy">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Купить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_DeleteFromFavorites">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить из избранного</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="mnu_menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="mnu_show_games"/>
    <addaction name="mnu_show_favorites"/>
    <addaction name="mnu_show_store"/>
    <addaction name="mnu_present"/>
    <addaction name="mnu_update_balance"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню разработчика</string>
    </property>
    <addaction name="mnu_create_game"/>
    <addaction name="mnu_update_game"/>
    <addaction name="mnu_update_file"/>
   </widget>
   <addaction name="mnu_menu"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="mnu_show_games">
   <property name="text">
    <string>Игры</string>
   </property>
  </action>
  <action name="mnu_show_favorites">
   <property name="text">
    <string>Избранное</string>
   </property>
  </action>
  <action name="mnu_show_store">
   <property name="text">
    <string>Магазин</string>
   </property>
  </action>
  <action name="mnu_create_game">
   <property name="text">
    <string>Добавить игру</string>
   </property>
  </action>
  <action name="mnu_update_game">
   <property name="text">
    <string>Изменить игру</string>
   </property>
  </action>
  <action name="mnu_update_file">
   <property name="text">
    <string>Добавить файлы игры</string>
   </property>
  </action>
  <action name="mnu_present">
   <property name="text">
    <string>Подарить игру</string>
   </property>
  </action>
  <action name="mnu_update_balance">
   <property name="text">
    <string>Пополнить баланс</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GameStoreGUI.qrc"/>
 </resources>
 <connections/>
</ui>
